docs/content/en/tasks/migrate/_index.md

```diff
@@ -6,227 +6,245 @@ weight: 1
 
-Before installing K8ssandra, make a note of the IP addresses of the seed nodes of the current datacenter. You will use the following to configure the `additionalSeeds` property in the k8ssandra chart. Here is an example chart properties overrides file named `k8ssandra-values.yaml` that we can use to install k8ssandra:
+## Create a K8ssandraCluster resource
+The cluster is now ready for the expansion to a new datacenter created by k8ssandra-operator.
 
-```yaml
-#k8ssandra-values.yaml
-#
-# Note this only demonstrates usage of the additionalSeeds property.
-# relrefer to the chart documentation for other properties you may want
-# to configure.
-cassandra:
-  # The cluster name needs to match the cluster name in the original
-  # datacenter.
-  clusterName: cassandra
-  datacenters:
-  - name: dc2
-    size: 3
-  additionalSeeds:
-  # The following should be replaced with actual IP addresses or
-  # hostnames of pods in the original datacenter.
-  - <dc1-seed>
-  - <dc1-seed>
-  - <dc1-seed>
+We’ll need to create a `K8ssandraCluster` (k8c) object with the right settings and secret references. Here’s an example `k8c` object with Reaper, Medusa and Stargate enabled:
+
+```
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: my-cluster
+spec:
+  reaper:
+    autoScheduling:
+      enabled: true
+    cassandraUserSecretRef:
+      name: reaper-secret
+    jmxUserSecretRef:
+      name: reaper-jmx-secret
+    uiUserSecretRef:
+      name: reaper-ui-secret
+  cassandra:
+    serverVersion: "4.0.1"
+    resources:
+      requests:
+        memory: 16Gi
+    additionalSeeds:
+      - "10.10.1.1"
+      - "10.10.1.2"
+    datacenters:
+      - metadata:
+          name: dc2
+        size: 3
+        storageConfig:
+          cassandraDataVolumeClaimSpec:
+            storageClassName: standard
+            accessModes:
+              - ReadWriteOnce
 ```
 
-Install k8ssandra as follows. Replace `my-k8ssandra` with whatever name you prefer:
+`metadata.name` needs to match the existing Cassandra cluster name so that the datacenters can connect together.
+
+The `additionalSeeds` section requires the IPs of a couple Cassandra nodes (or pods) from the existing datacenter. Those IPs must be reachable on port 7001, which is Cassandra’s storage port used by nodes to communicate with each other.
+
+K8ssandra-operator automatically manages the replication strategy for several keyspaces, such as `system_auth`, `data_endpoint_auth` or `reaper_db`, based on the list of datacenters in a k8c object. It allows safe automation for operations such as expansions to new datacenters. A migration involves one or more datacenters which are not referenced in the k8c object, requiring the addition of extra datacenters in the replication settings. This is what the `externalDatacenters` entry stands for, allowing us to keep replicas on the datacenters we are migrating from.
```

Comment:
I don't  see any `externalDatacenters` entry? This paragraph confused me.

Reply:
right, that's due to the above bug that truncated the spec.
It'll be fixed in the next commit.